### DISCUSSION TITLE
Naomi wipi netbooting

### DISCUSSION THREAD

**Discussion by rogersta99** [retronas/discussions#87] [Ideas]
Is it possible to integrate the Naomi wipi netbooting, https://www.arcade-projects.com/threads/naomi-pi-wipi-netbooter-compatible-with-pi-3-4.14019/ within the retronas structue.  The most recent releases have support for connecting to wifi networks.

**Comment by sairuk** [comment]
Can you provide some detail here on this, that link requires an account.

**Reply by rogersta99** [reply]
Also here on github:  https://github.com/chunksin/WiPiNetbooter

Features:
Memory check bypass for faster game loading
Time hack - no need to use a zero pic on a Naomi/Naomi 2
Multiple netdimm support per system (Naomi, Naomi 2, Triforce and Chihiro)
Hotspot Wifi and Home Wifi/Wired network support with DHCP or static IP addressing
Netdimm firmware upgrade for type 1
16 x 2 LCD and 3.5 inch touchscreen support
Single boot mode for automatic game loading
Compressed game roms so everything fits onto a 32GB micro SD card (SanDisk 32GB recommended)
Curated, standardised romset thanks to [@Murray](https://www.arcade-projects.com/members/6252/)
OpenJVS support thanks to [@bobbydilley](https://www.arcade-projects.com/members/3985/) (requires additional adaptor)
Beta OpenFFB support thanks to [@Fredobedo007](https://www.arcade-projects.com/members/2141/) (requires additional adaptor)
Card reader emulator for Initial D, Wangan Midnight and F-Zero games thanks to [@winteriscoming](https://www.arcade-projects.com/members/605/) (requires additional adaptor)
NFC Card support for card emulator save games
Card printing in the web interface for Initial D, Initial D2, Initial D3 and F-Zero AX
On screen menu for choosing and loading games that runs on the Naomi/Naomi 2 thanks to [@DragonMinded](https://www.arcade-projects.com/members/5648/)
No more triforcetools! backend now uses [@DragonMinded](https://www.arcade-projects.com/members/5648/)'s netboot suite
Server mode for automatic and unattended netbooting
Update function added so no more giant downloads

**Reply by sairuk** [reply]
I've added it to the [ideas](https://github.com/danmons/retronas/wiki/Ideas) page for now, 

I took a preliminary look and my initial thoughts are 
- the project layout is all over the shop so it may be difficult (not impossible) to incorporate
- need to think how we incorporate it with existing tooling (lighttpd)
- It looks to be a python3 tool so that is a bonus.

thanks for the suggestion, neither @danmons nor myself currently have the hardware required to test this kind of tool unfortunately so we'd be going in blind.
